scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,853 @@
+platform_8 = ['x86_64-mlnx_msn4600c-r0','x86_64-mlnx_msn4700-r0']
+
+bko_dict_4 = {
+        "1x100": { "lanes":4, "speed":100000, "step":4, "bko":0, "name": "etp" },
+        "1x40":  { "lanes":4, "speed":40000,  "step":4, "bko":0, "name": "etp" },
+        "1x50":  { "lanes":4, "speed":50000,  "step":4, "bko":0, "name": "etp" },
+        "1x25":  { "lanes":4, "speed":25000,  "step":4, "bko":0, "name": "etp" },
+        "1x10":  { "lanes":4, "speed":10000,  "step":4, "bko":0, "name": "etp" },
+        "1x1":   { "lanes":4, "speed":1000,   "step":4, "bko":0, "name": "etp" },
+        "4x10":  { "lanes":4, "speed":10000,  "step":1, "bko":1, "name": "etp" },
+        "4x25":  { "lanes":4, "speed":25000,  "step":1, "bko":1, "name": "etp" },
+        "2x50":  { "lanes":4, "speed":25000,  "step":2, "bko":1, "name": "etp" },
+}
+
+bko_dict_8 = {
+        "1x400": { "lanes":8, "speed":400000, "step":8, "bko":0, "name": "etp" },
+        "2x200": { "lanes":8, "speed":200000, "step":4, "bko":1, "name": "etp" },
+        "2x100": { "lanes":8, "speed":100000, "step":4, "bko":1, "name": "etp" },
+        "4x100": { "lanes":8, "speed":100000, "step":2, "bko":1, "name": "etp" },
+        "4x50":  { "lanes":8, "speed":50000,  "step":2, "bko":1, "name": "etp" },
+        "4x25":  { "lanes":4, "speed":25000,  "step":1, "bko":1, "name": "etp" },
+        "4x10":  { "lanes":4, "speed":10000,  "step":1, "bko":1, "name": "etp" },
+}
+
+bko_dict = bko_dict_4
+
+class SkuCreate(object):
+        """
+	Process aclstat
+	"""
+
+	def __init__(self):
+
+		self.portconfig_dict = {}
+		self.platform_specific_dict = {"x86_64-mlnx_msn2700-r0":self.msn2700_specific}
+		self.default_lanes_per_port = []
+                self.platform = None
+                self.base_lanes = None
+		self.fpp = []
+		self.fpp_split  =  {}
+		self.num_of_fpp = 0
+		self.num_of_lanes_dict = {"1" : 4, "2" : 2, "4" : 1}
+		self.sku_name = None
+                self.default_sku_path = None
+		self.base_sku_name = None
+		self.base_sku_dir = None
+		self.base_file_path = None		
+		self.new_sku_dir = None
+		self.print_mode = False
+		self.remove_mode = False
+		self.verbose = None
+
+	def read_metadata(self):
+		"""
+		Redis database platform info
+		"""
+		try:
+			self.metadata = self.db.get_all(self.db.CONFIG_DB, "DEVICE_METADATA|localhost" )
+		except:
+			print ("Error While trying to retrieve METADATA from CONFIG_DB", file=sys.stderr)
+			exit(1)
+			
+	def sku_def_parser(self,sku_def) :
+		# Parsing XML sku definition file to extract Interface speed and InterfaceName(alias) <etp<#><a/b/c/d> to be used to analyze split configuration
+		# Rest of the fields are used as placeholders for portconfig_dict [name,lanes,SPEED,ALIAS,index]
+		try:
+			f = open(str(sku_def),"r")
+		except IOError:
+			print ("Couldn't open file: " + str(sku_def), file=sys.stderr)
```

Comment:
@madhanmellanox no space: `print(<stuff_goes_here>)`

Reply:
Addressed in next commit.